content/apidocs-mxsdk/apidocs/team-server-api.md

```diff
@@ -4,203 +4,411 @@ category: "API Documentation"
+
+Payload Example:
+```
+{
+    "errorCode": "RS400",
+    "errorMessage": "Please provide valid input to execute this request. Invalid project id"
+}
+```
 
-## 3 API Calls
+## 4 API Calls
 
-### 3.1 Retrieve Branches
+### 4.1 Retrieve Repository Info
 
-<a name="TeamServerAPI-Description" rel="nofollow"></a>Retrieves all branches that belong to the team server project of a specific app which the authenticated user has access to as a regular user.
+<a name="TeamServerAPI-Description" rel="nofollow"></a>Retrieve information about the version control repository for a Mendix app.
 
 ```http
 HTTP Method: GET
- URL: https://deploy.mendix.com/api/1/apps/<AppId>/branches
+ URL: https://teamserver.api.mendix.com/v1/repositories/<appId>
 ```
 
-#### 3.1.1 Request
+#### 4.1.1 Request
 
-##### 3.1.1.1 Parameter
+##### 4.1.1.1 Path Parameters
 
-*   _AppId_ (String) : Subdomain name of an app.
+|Name|Type|Required|Description|
+|---|---|---|---|
+|`AppId`|String|Yes|The app ID (also known as project ID) of the Mendix project for which the repository information should be returned.|
 
-##### 3.1.1.2 Example
+##### 4.1.1.2 Example
 
 ```http
-GET /api/1/apps/calc/branches HTTP/1.1
-Host: deploy.mendix.com
-
+GET /v1/repositories/c0af1725-edae-4345-aea7-2f94f7760e33 HTTP/1.1
+Host: teamserver.api.mendix.com
 Accept: */*
-Mendix-Username: richard.ford51@example.com
-Mendix-ApiKey: 26587896-1cef-4483-accf-ad304e2673d6
+Authorization: MxToken hZUPhAV4ELPrRm7U7JAKf5BnxJk6q7dcsvFdw6ZR4wRYdv7egHjwHEYBwXY4RkSZrAWde3XqVAQkxZNPysvHcpquA9sK9bsKmcTN
 ```
 
-#### 3.1.2 Output
+#### 4.1.2 Response Headers
 
-List of objects with the following key-value pairs:
+|Name|Value|
+|---|---|
+|Content-Type|`application/json; charset=utf-8`|
 
-*   _Name_ (String) : Name of the branch. This is 'trunk' for the main line or a specific branch name.
-*   _DisplayName_ (String) : Visible name in the Developer Portal. For the trunk, this is 'Main line'.
-*   _LatestRevisionNumber_ (Long) : Number of the latest revision.
-*   _LatestRevisionMendixVersion_ (String) : Version string of the Mendix version of the app in this revision.
+#### 4.1.3 Response Payload
 
-##### 3.1.2.1 Error Codes
+List of objects with the following key-value pairs:
 
-| HTTP Status | Error code | Description |
-| --- | --- | --- |
-| 400 | INVALID_APPID | Invalid AppId. |
-| 404 | APP_NOT_FOUND | App not found. |
+|Name|Type|Description|
+|---|---|---|
+|`projectId`|String|The project ID (also known as app ID) of the Mendix project.|
+|`type`|String|The type of repository. At the moment this will be either `"svn"` or `"git"`, but later on other repository types may be introduced.|
+|`url`|String|The URL of the repository.|
+|`isMendixHosted`|Boolean|Whether the repository is Mendix-hosted (i.e. a Team Server repository) or is using a custom repository URL. Note: If a repository is not Mendix-hosted (i.e. `isMendixHosted` is `false`), the other operations of the Team Server API will not work for this repository.|
```

Comment:
Apply the note style:
{{% alert type="note" %}}
If a repository is not Mendix-hosted (i.e. `isMendixHosted` is `false`), the other operations of the Team Server API will not work for this repository.|
{{% /alert %}}

Reply:
We removed `isMendixHosted` field from the API, this warning is now removed from the docs